android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -523,6 +536,21 @@ Call.Issue getIssueFromString(String issue) {
     }
     return Call.Issue.NOT_REPORTED;
   }
+
+  CallMessage.MessageType getMessageTypeFromString (String messageType) {
```

Comment:
Is this method private? is it being called anywhere outside the class? if not, lets make it private. Also maybe change the name to validate message type.. also, instead of silently failing, should we report some error to the user, maybe a js exception or something?